README.md

```diff
@@ -121,81 +95,71 @@ Also in the output will be the endpoints that will be used to interface with the
 The default ws url is "127.0.0.1:8546" and the default http url is "127.0.0.1:8545".
 These values will be used as the `ethereum.wsPath` and `ethereum.httpPath` in the config, respectively.
 
-#### Bitcoin
-For Bitcoin, ipfs-blockchain-watcher is able to operate entirely through the universally exposed JSON-RPC interfaces.
-This means any of the standard full nodes can be used (e.g. bitcoind, btcd) as the data source.
+### Indexer
+Finally, setup the indexer process itself.
 
-Point at a remote node or set one up locally using the instructions for [bitcoind](https://github.com/bitcoin/bitcoin) and [btcd](https://github.com/btcsuite/btcd).
+Start by downloading ipld-eth-indexer and moving into the repo:
 
-The default http url is "127.0.0.1:8332". We will use the http endpoint as both the `bitcoin.wsPath` and `bitcoin.httpPath`
-(bitcoind does not support websocket endpoints, the watcher currently uses a "subscription" wrapper around the http endpoints)
+`GO111MODULE=off go get -d github.com/vulcanize/ipld-eth-indexer`
```

Comment:
Don't we want users to download binary from releases instead of building them?

Reply:
Yes definitely, we need to integrate GitHub actions into this repository first. I think that is a separate issue/PR of its own, one that I'd like to assign to you since you've been going through that process with the other repos.